django_rq/settings.py

```diff
@@ -25,3 +25,8 @@
 
 # Token for querying statistics
 API_TOKEN = getattr(settings, 'RQ_API_TOKEN', '')
+
+# Activate locale for each task
+RQ_USE_LOCALIZATION = getattr(settings, 'RQ_USE_LOCALIZATION', False)
```

Comment:
Should we rename this to `RQ_USE_L10N` to keep it consistent with Django's naming scheme? `RQ_LANGUAGE_CODE` defaults to Django's `LANGUAGE_CODE` setting, should we also change this to default to Django's `USE_L10N` setting?